spring-petclinic-genai-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,43 @@
+spring:
+  main:
+    web-application-type: reactive
+  application:
+    name: genai-service
+  profiles:
+    active: production
+  config:
+    import: optional:configserver:${CONFIG_SERVER_URL:http://localhost:8888/},optional:classpath:/creds.yaml
+  #These apply when using spring-ai-azure-openai-spring-boot-starter
```

Comment:
I'm not sure if we can reliably export the credentials in the config server. Since this is private sensitive data, and the config server is publicly available on Github, the creds.yaml should remain in the source code of the genai service. It is listed for ignore in .gitignore.

Reply:
The config repository could be cloned in a private repository. It's URL could be provided with the `CONFIG_SERVER_URL` environment variable. I agree: it's not very practical.

I'm not very fan of adding the `creds.yaml` in a Docker image. We could use a Docker volume. But I'll prefer to use an environment variable like `AZURE_OPENAI_KEY` and `AZURE_OPENAI_ENDPOINT`. See https://github.com/spring-petclinic/spring-petclinic-ai/commit/fbaf8dc283b8a70588aa655921411dd838c681c8#diff-54eeffbae371fcd1398d4ca5e89a1b8118208b7bb2f8ddf55c1aa2f7d98ab136R36
Developers can pass those parameteres from their IDE or from the Docker command line. What do you think about?

Reply:
Yep, that would work.